lisa/notifier.py

```diff
@@ -40,6 +42,59 @@ class TestRunMessage(MessageBase):
     message: str = ""
 
 
+@dataclass
+class PerfMessage(MessageBase):
+    pass
+
+
+DiskSetupType = Enum(
+    "DiskSetupType",
+    [
+        "raw",
+        "raid0",
+    ],
+)
+
+
+DiskType = Enum(
+    "DiskType",
+    [
+        "nvme",
+        "premiumssd",
+    ],
+)
+
+
+@dataclass
+class DiskPerformanceMessage(PerfMessage):
+    tool: str = constants.DISK_PERFORMANCE_TOOL
+    test_case_name: str = ""
+    platform: str = ""
+    location: str = ""
+    host_version: str = ""
+    guest_os_type: str = "Linux"
+    distro_version: str = ""
+    vmsize: str = ""
+    kernel_version: str = ""
+    lis_version: str = ""
+    disk_setup_type: DiskSetupType = DiskSetupType.raw
+    block_size: int = 0
+    disk_type: DiskType = DiskType.nvme
+    core_count: int = 0
```

Comment:
Please save the number of jobs too. It's clearer. And we may try different number of jobs in future.